raiden-ts/docs-source/SDK-Development.md

```diff
@@ -226,23 +227,23 @@ action.pipe(
+The unit tests try to cover as much as possible the individual functions, by testing behavior through expected and unexpected inputs, and their respective outputs. For that, we use extensively [jest mocks](https://jestjs.io/docs/en/mock-functions) to contextually replace external and internal dependencies of each tested function, and force the external logic to behave on the different (possible, conceivable) ways and ensure the our tested logic handles all of them. Most of the unit mocks are in [raiden/tests/unit/mocks.ts](https://github.com/raiden-network/light-client/blob/master/raiden-ts/tests/unit/mocks.ts), but some others are needed to be put in the beginning of the respective test files, for the mocking to take place before importing code using the mocked logic.
 
 We try to split unit tests by kind of tested function. Most of the tested functions are pure (like `utils`, or `reducers`) or have very explicit dependencies (`epics`). To add a new test, simply add the `describe` and `test` calls to the respective function type.
 
 The hardest to unit test are the epics. As they conceive most of the Raiden logic, are async and may depend on multiple parts working together (e.g. an output action changing input state), up to now we don't have a standardized way of writting their tests, although as we look to writting them in the most isolated way possible and each having a single codepath (rxjs operators pipeline) helps into wrapping and setting up all required testing context to get maximum coverage and ensuring the different scenarios are properly handled. `Observable.toPromise()` and [rxjs-marbles](https://cartant.github.io/rxjs-marbles/) can help into inspecting the outputs.
 
 ### e2e tests
 
-e2e tests try to test the SDK as well as the dependency stack from the [public API](https://github.com/raiden-network/light-client/blob/master/raiden/src/raiden.ts) perspective, by performing proper actions as a user would do. Adding tests here basically involve adding them to [raiden.spec.ts](https://github.com/raiden-network/light-client/blob/master/raiden/tests/e2e/raiden.spec.ts), with required Matrix endpoints and state.
+e2e tests try to test the SDK as well as the dependency stack from the [public API](https://github.com/raiden-network/light-client/blob/master/raiden-ts/src/raiden.ts) perspective, by performing proper actions as a user would do. Adding tests here basically involve adding them to [e2e.spec.ts](https://github.com/raiden-network/light-client/blob/master/raiden-ts/tests/e2e/e2e.spec.ts), with required Matrix endpoints and state.
```

Comment:
Notice that what we called `e2e` is now the `integration` tests, and e2e is a new test suite using cypress.

Reply:
Mh, but what about the `raiden-ts/tests/e2e` folder?